server/coordinator.go

```diff
@@ -165,27 +166,63 @@ func (c *coordinator) run() {
 	}
 	log.Info("coordinator starts to run schedulers")
 
-	k := 0
+	scheduleNames, configs, err := c.cluster.storage.LoadAllScheduleConfig()
+	if err != nil {
+		log.Fatal("cannot load schedulers' config")
```

Comment:
Is this necessary to use fatal here?

Reply:
I think we have to let the scheduler run normally, otherwise, some users may ignore the problem that the schedule is now work.